src/CurrencyHelper.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace PodPoint\I18n;
+
+use Illuminate\Config\Repository;
+use NumberFormatter;
+
+class CurrencyHelper extends Helper
+{
+    /**
+     * Instance of the CountryHelper.
+     *
+     * @var CountryHelper
+     */
+    protected $countryHelper;
+
+    /**
+     * @param Repository $config
+     */
+    public function __construct(Repository $config)
+    {
+        parent::__construct($config);
+
+        $this->countryHelper = new CountryHelper($this->config);
+    }
+
+    /**
+     * Return a value formatted in the right currency and locale.
+     *
+     * @param float $value
+     * @param string $locale
+     * @param string $currencyCode
+     *
+     * @return string
+     */
+    public function toFormat(float $value, string $locale = 'en', string $currencyCode = 'GBP')
```

Comment:
I presume the default values can't use the constants?

Reply:
Will this work ok if the value is an integer rather than a float?

Reply:
I'm going to test both 👍

Reply:
-So we can use Constant as default value which I wasn't aware of
-and we can pass `int` and `float` without any problem so I'm going to remove the float type hinting of $value.